src/silx/math/fit/fittheories.py

```diff
@@ -1263,6 +1354,13 @@ def configure(self, **kw):
                               'HighFWHM', 'Eta'),
                   estimate=fitfuns.estimate_splitpvoigt,
                   configure=fitfuns.configure)),
+    ('Split Pseudo-Voigt 2',
+        FitTheory(description='Asymmetric pseudo-Voigt functions',
```

Comment:
Maybe have a different `description` that the one for `functions.sum_splitpvoigt`? :shrug:

Reply:
- `Asymmetric pseudo-Voigt functions with split Lorentzian factor`
- `Asymmetric pseudo-Voigt functions with two Lorentzian factors`
- ???

Reply:
I went for 'Asymmetric pseudo-Voigt functions with split Lorentzian fraction'